src/sorcha_cmdline/config.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+import sys
+
+#
+# sorcha config prepare
+#
+
+def parse_file_selection(file_select):
+    """Turns the number entered by the user at the command line into a string
+    prompt. Also performs error handling.
+
+    Parameters
+    -----------
+    file_select : int
+        Integer entered by the user at command line.
+
+    Returns
+    -----------
+    which_configs : string
+        String indicating which configuration files to retrieve. Should be "rubin", "demo" or "all".
+
+    """
+
+    try:
+        file_select = int(file_select)
+    except ValueError:
+        sys.exit("Input could not be converted to a valid integer. Please try again.")
+
+    if file_select not in [1, 2, 3]:
+        sys.exit("Input could not be converted to a valid integer. Please input an integer between 1 and 3.")
+
+    selection_dict = {1: "rubin_circle", 2: "rubin_footprint", 3: "all"}
+
+    which_configs = selection_dict[file_select]
+
+    return which_configs
+
+def cmd_prepare(args):  # pragma: no cover
+    print("\nWhich configuration file(s) would you like to copy?:\n")
+    print("1. Rubin-specific configuration file using circular approximation of camera footprint (faster).\n")
+    print("2. Rubin-specific configuration file using full camera footprint (slower, but more accurate).\n")
+    print("3. All.\n")
+    file_select = input("Please enter a number and hit Return/Enter.\n")
+
+    which_configs = parse_file_selection(file_select)
+
+    #
+    # NOTE: DO NOT MOVE THESE IMPORTS TO THE TOP LEVEL OF THE MODULE !!!
+    #
+    #       Importing sorcha from the function and not at the top-level of the module
+    #       allows us to exit quickly and print the help/error message (in case there
+    #       was a mistake on the command line). Importing sorcha can take 5 seconds or
+    #       more, and making the user wait that long just to print out an erro message
+    #       is poor user experience.
+    #
+    from sorcha.utilities.sorcha_copy_configs import copy_demo_configs
+    import os
+
+    copy_location = os.path.abspath(args.path)
+    return copy_demo_configs(copy_location, which_configs, args.force)
+
+#
+# sorcha config
+#
+
+def main():
+    # Create the top-level parser
+    parser = argparse.ArgumentParser(prog="sorcha-config", description="Sorcha Configuration Utility")
+    subparsers = parser.add_subparsers(title="commands", description="Available commands", help="Command to execute", dest='command')
+
+    # Add the `prepare` subcommand
+    prepare_parser = subparsers.add_parser("prepare", help="Prepare configuration")
```

Comment:
I'm in favor of prepare because I'm thinking of it as preparing a configuration file to run some kind of a simulation. Whether it's generated or fetched from somewhere is irrelevant to the user -- what it does from their perspective is preparing the environment (config, and possibly other things in the future -- like data files) for a sorcha run.

Reply:
can we add a "prepare default configuration"  or "prepare default configuration"

Reply:
Hmm, there's no true "default" -- there are already two configs to choose from (and as more instruments are added, more may appear).

Thinking more about this, how about using `init` here? I.e.: `sorcha config init` initializes the configuration files for a simulation run.

Even better, I can drop `config` (i.e., only have `sorcha init`). This would make a typical workflow look like:
```
sorcha init
... fiddle with config files ...
sorcha run
```
I like this latter option (also saves 5 characters of typing :) )...

Reply:
I like init

Reply:
👍. Give me five minutes...